mmv1/products/vertexai/FeatureOnlineStoreFeatureview.yaml

```diff
@@ -118,3 +124,62 @@ properties:
         description: |
           Columns to construct entityId / row keys. Start by supporting 1 only.
         item_type: Api::Type::String
+  - !ruby/object:Api::Type::NestedObject
+    name: 'vectorSearchConfig'
+    description: |
+      Configuration for vector search. It contains the required configurations to create an index from source data, so that approximate nearest neighbor (a.k.a ANN) algorithms search can be performed during online serving.
+    immutable: true
+    min_version: beta
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'embeddingColumn'
+        required: true
```

Comment:
according to the [API docs](https://cloud.google.com/vertex-ai/docs/reference/rest/v1beta1/projects.locations.featureOnlineStores.featureViews#VectorSearchConfig) this is optional but the description of the field says it's required, so I assume this is correct?

Reply:
Yeah, It is required. I've tried making a request to create this resource without the embedding column and recieved 400 errors with message "`embedding_column` must be set to use vector search.". I've raised a bug with the team for updating the API documentation.